src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
I can understand, my motivation was that this would save others from having to implement the same generic wrapper.